sirepo/package_data/static/js/sirepo-components.js

```diff
@@ -1871,6 +1871,51 @@ SIREPO.app.directive('validatedString', function(panelState, validationService)
     };
 });
 
+SIREPO.app.directive('viewLogIframe', function() {
+    return {
+        restrict: 'A',
+        scope: {
+            loadingLog: '<',
+            logHtml: '<',
+            logPath: '<',
+            wrapperDownloadLog: '<'
+        },
+        template: `
+            <div class="modal fade" id="sr-iframe-text-view" tabindex="-1" role="dialog">
```

Comment:
I think the id should be "sr-view-log-iframe" since that is the name of the directive.

Reply:
4d8ecf81fb4e98997af2c3b6de70315095700cc9